### PR TITLE
[LJI-4] Manual to run project locally

### PR DESCRIPTION
**Pull request by NormantasN**
Removed redundant dependencies and configurations, changed config from postgres to H2

### CODE REVIEW

File: src/main/resources/db/changelog/changelog-master.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
```
Comment (anton-kolyaev) [inline src/main/resources/db/changelog/changelog-master.xml:2]: Please move each changeset to it's own file and keep root one only to include files. 
https://docs.liquibase.com/change-types/include.html

**Review by anton-kolyaev — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/changelog-1.xml
Comment (anton-kolyaev) [inline src/main/resources/db/changelog/changelog-1.xml:1]: 1. do we need it at all in this commit?
2. please follow filenaming template as here
https://github.com/Java-POT-Summer-24/pot-java-summer-24/pull/8/files#diff-e42ff0f1696d9ae282dde9fb16962b555d9c4c940e435315916046a4ac8a54d7

**Review by anton-kolyaev — CHANGES_REQUESTED** [review]

File: src/main/resources/db/changelog/changelog-1.xml
Comment (NormantasN) [reply src/main/resources/db/changelog/changelog-1.xml:1]: Done

File: src/main/resources/db/changelog/changelog-master.xml (hunk 1 above)
Comment (NormantasN) [reply src/main/resources/db/changelog/changelog-master.xml]: Done

File: build.gradle [hunk 2]
```diff
@@ -19,8 +19,18 @@ repositories {
 
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter'
+	implementation 'org.springframework.boot:spring-boot-starter-security'
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+	testImplementation group: 'com.h2database', name: 'h2', version: '1.3.148'
```
[the remark this answers is not in the document]
Comment (NormantasN) [reply build.gradle]: Done

**Review by SavaLX — APPROVED** [review]

**Review by anton-kolyaev — APPROVED** [review]
resolve conflicts and you're welcome to merge it

**Pull request merged**